vault/resource_kubernetes_auth_backend_config_test.go

```diff
@@ -352,13 +402,20 @@ resource "vault_auth_backend" "kubernetes" {
 resource "vault_kubernetes_auth_backend_config" "config" {
   backend = vault_auth_backend.kubernetes.path
   kubernetes_host = "http://example.com:443"
-  kubernetes_ca_cert = %q
   token_reviewer_jwt = %q
-}`, backend, kubernetesCAcert, jwt)
+`, backend, jwt)
+
+	if ca != "" {
+		config += fmt.Sprintf(`
+  kubernetes_ca_cert = %q
+`, ca)
+	}
```

Comment:
```suggestion
	} else {
		config += fmt.Sprintf(`
pem_keys = %q
`, []string{kubernetesPEMfile})
	}
```

Reply:
Resolving since this looks to be fixed in v1.9.3

Reply:
Seems to break when testing against `1.9.{0..2}+ent`, but works fine on `1.9.3+ent`